storages/python/tests-python/test_mongodb-atlas_python.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import os
+from pathlib import Path
+import dlite
+
+# Load user and password from environment variables
+user = os.environ.get("DLITETEST_MONGODB_USER")
+password = os.environ.get("DLITETEST_MONGODB_PASSWORD")
+server = os.environ.get("DLITETEST_MONGODB_SERVER")
+inputdir = os.environ.get("DLITETEST_MONGODB_INPUTDIR")
```

Comment:
Setting the credentials in the environment is one way of providing them. But isn't it be more convenient to ask the user to add a local file with the credentials? That way, if you once have set up the credentials, it will run whenever you run the tests, without any need to set up the environment.

The file with credentials should of cause be listed in .gitignore to avoid that it by mistake is committed.

This is how we do in the C tests for postgresql.

Reply:
The test is now robust and will try to use either environment variables or a file.